### PR TITLE
Removing deprecated --with-natural-title-bar option.

### PR DESCRIPTION
**Pull request by jwintz**
And added note for the sake of history and clarity.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -75,6 +73,14 @@ option at your own risk. And feel free to open up issues you face, just remember
 that I am not close enough to Emacs development process to provide you much help
 with any arising issues.
 
+Note that =--with-natural-title-bar= has been deprecated since it is
+available for =--HEADS= builds using the following lisp snippet
```
Comment (d12frosted) [inline README.org]: Shouldn't it be `--HEAD` instead of `--HEADS`? 😸

**Comment by jwintz** [comment]
You’re absolutely right, I’ll fix the typo asap, i’ll be automatically added to the PR.

  - ju

**Comment by jwintz** [comment]
Done.

  - ju

**Comment by d12frosted** [comment]
Thanks 😸

**Comment by jwintz** [comment]
BTW, Are you still willing to put a screenshot on the README ?

If so I can provide some.

  - ju

**Comment by j-martin** [comment]
Would it be possible to wait until the natural title bar is available in a stable release before removing this flag? I am not really interested in running HEAD, but I do love the natural title bar.

**Comment by d12frosted** [comment]
> BTW, Are you still willing to put a screenshot on the README ?
>
> If so I can provide some.

Yes, I do 😸 Would be nice to have some.

> Would it be possible to wait until the natural title bar is available in a stable release before removing this flag? I am not really interested in running HEAD, but I do love the natural title bar.

This is what I wanted to check before merging - if the functionality is available in the master branch. Seems like it is not, so I think it would be good to wait a little bit, before new Emacs release. 

@jwintz what do you think? 😸

**Comment by jwintz** [comment]
Boris <notifications@github.com> writes:

>> BTW, Are you still willing to put a screenshot on the README ?
>>
>> If so I can provide some.
>
> Yes, I do 😸 Would be nice to have some.
>
>> Would it be possible to wait until the natural title bar is available in a stable release before removing this flag? I am not really interested in running HEAD, but I do love the natural title bar.
>
> This is what I wanted to check before merging - if the functionality is available in the master branch. Seems like it is not, so I think it would be good to wait a little bit, before new Emacs release.
>
> @jwintz what do you think? 😸

No problem, the gist is still online, I'll keep the PR as it is as long
as needed.

Cheers,

  - ju

**Comment by j-martin** [comment]
While I am here: screen shot 2017-11-16 at 09 23 36

**Comment by d12frosted** [comment]
@j-martin thanks for the screenshot

I've added it to readme 😸 

@jwintz I have organised all the options a bit. Sorry, but your PR is broken now. 😢  If it's not hard, could you please fix conflicts?

**Comment by shishirsharma** [comment]
bump!

**Comment by d12frosted** [comment]
@shishirsharma this PR is waiting for the master branch of Emacs to include this functionality. AFAIK it's not there yet and so this PR is here :) Let me know if I am missing something

**Comment by d12frosted** [comment]
Sorry, I could not get the change from you as you have already deleted the fork.

**Pull request closed without merging**